Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -495,4 +574,59 @@ public class ConnackPacket {
         self.responseInformation = responseInformation
         self.serverReference = serverReference
     }
+
+    static func convertFromNative(_ from: UnsafePointer<aws_mqtt5_packet_connack_view>?) -> ConnackPacket? {
+
+        if let from = from {
+
+            let sessionPresent = from.pointee.session_present
+            let reasonCode = ConnectReasonCode(rawValue: Int(from.pointee.reason_code.rawValue)) ?? .unspecifiedError
```

Comment:
We use `fatalError` in `DisconnectPacket` when the error code is invalid. Would we like to do the same thing here?